### PR TITLE
Change fetcher of my packages

### PR DESCRIPTION
**Pull request by rubikitch**
emacswiki to github

### CODE REVIEW

File: recipes/git-dwim [hunk 1]
```diff
@@ -1 +1 @@
-(git-dwim :fetcher wiki)
+(git-dwim :repo "rubikitch/emacs-git-dwim" :fetcher github)
```
Comment (milkypostman) [inline recipes/git-dwim]: i'm not sure how 50git-dwim will do here. but i'm willing to give it a try.

**Comment by milkypostman** [comment]
a bit different in a few cases but overall no major changes.

File: recipes/git-dwim (hunk 1 above)
Comment (purcell) [inline recipes/git-dwim]: Yes, that should be removed. It's personal configuration which shouldn't be distributed.

**Comment by tarsius** [comment]
@rubikitch Thanks for doing this!

I've been mirroring these packages on the [Emacsmirror](https://emacsmirror.net/) (please read the first section, including the part below the image) and have now changed that to get these package from your repositories.

The Emacsmirror also contains some additional packages whose author you are. Do you plan to move these to Github too? Are they still functional? Should they be moved from the mirror to the Emacsattic?

**Comment by tarsius** [comment]
- [ ] anything-kyr
- [ ] anything-kyr-config
- [ ] auto-install
- [ ] d-insert-assert
- [ ] d-insert-import
- [ ] display-buffer-for-wide-screen
- [ ] display-hatebu
- [ ] eijiro
- [ ] el-expectations
- [ ] eldoc-extension
- [ ] emacs-init-check
- [ ] ert-expectations
- [ ] icicles-iswitchb
- [ ] install-elisp
- [ ] lispxmp
- [ ] minor-mode-hack
- [ ] mode-line-frame
- [ ] recentf-ext
- [ ] screenshot
- [ ] sequential-command
- [ ] shell-history
- [ ] space-chord
- [ ] sticky
- [ ] unify-buffer
- [ ] usage-memo

**Pull request merged**